Document Android Fragment perf. (#3805)
getsentry/sentry-docs: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

When you didn't finish the transaction yet, but you start a new Activity, the SDK always automatically finishes the previous Activity transaction. This is due to the fact that only one transaction at a time can be bound to the Scope. To work around that, you may create transactions manually using the custom instrumentation and its instance to start spans instead of the Static API.

> ### Android's Fragment's Instrumentation

Reviewer: "Fragment" doesn't need to be a possessive. 

```suggestion
### Android's Fragment Instrumentation
```

> The Fragment's instrumentation, once enabled, starts a span for each launch of a Fragment. The SDK sets the Span operation to `ui.fragment.load` and the Span description to the Fragment's name, for example, `LoginFragment`.

Reviewer: Android docs don't capitalize "fragment" after the first use of it (where it's a linked definition), so I've taken out the capital letter except in places where you use the full title again. I've also done the same with the word "span." 

```suggestion
Once enabled, Android's Fragment Instrumentation starts a span for each launch of a fragment. The SDK sets the span operation to `ui.fragment.load` and the span description to the fragment's name, for example, `LoginFragment`.
```

> The span starts after each Fragment's `onCreate` and before `onCreateView` method is called.

Reviewer: ```suggestion
The span starts after each fragment's `onCreate` method is called and before its `onCreateView` method is called.
```

> The span finishes after each Fragment's `onResume` method is executed.

Reviewer: ```suggestion
The span finishes after each fragment's `onResume` method is executed.
```

> The Android's Fragment's Instrumentation depends on having an active transaction bound to the scope, ideally it'd be used along with the [Android's Activity Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) which starts a transaction and bind it to the scope automatically.

Reviewer: ```suggestion
Android's Fragment Instrumentation depends on having an active transaction bound to the scope, and ideally it'd be used along with [Android's Activity Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation), which starts a transaction and binds it to the scope automatically.
```

> For more information see our [Fragment](/platforms/android/configuration/integrations/fragment/).

Reviewer: ```suggestion
Learn more in our [Fragment documentation](/platforms/android/configuration/integrations/fragment/).
```
